pkg/validation/internal/operatorhub.go

```diff
@@ -21,9 +21,51 @@ import (
 // OperatorHubValidator validates the bundle manifests against the required criteria to publish
 // the projects on OperatorHub.io.
 //
+// This validator will ensure that:
+// - The annotations capabilities into the CSV has a valid option, which are:
+// * "Basic Install"
+// * "Seamless Upgrades"
+// * "Full Lifecycle"
+// * "Deep Insights"
+// * "Auto Pilot"
+// - The annotations categories into the CSV has a valid option, which are:
+// * "AI/Machine Learning"
+// * "Application Runtime"
+// * "Big Data"
+// * "Cloud Provider"
+// * "Developer Tools"
+// * "Database"
+// * "Integration & Delivery"
+// * "Logging & Tracing"
+// * "Monitoring"
+// * "Networking"
+// * "OpenShift Optional"
+// * "Security"
+// * "Storage"
+// * "Streaming & Messaging"
+// - The csv.Spec.Provider.Name was provided
+// - The csv.Spec.Maintainers elements contains both name and email
+// - The csv.Spec.Links elements contains both name and url
+// - The csv.Spec.Links.Url is a valid value
+// - The csv.Spec.Version is provided
+// - The checks.csv.Spec.Icon was provided and has not more than one element
+// - The csv.Spec.Icon elements should contain both data and mediatype
+// - The csv.Spec.Icon elements should contain both data and mediatype
+// - The csv.Spec.Icon has a valid mediatype, which are
+// * "image/gif"
+// * "image/jpeg"
+// * "image/png"
+// * "image/svg+xml"
+// - If informed ONLY, check if the value csv.Spec.MinKubeVersion is parsable according to semver (https://semver.org/)
+// Also, this validator will raise warnings when:
+// - The bundle name (CSV.metadata.name) seems not following the naming convention: <operator-name>.v<semver> e.g. memcached-operator.v0.0.1
+// - The channel names seems are not following the convention https://olm.operatorframework.io/docs/best-practices/channel-naming/
+// - The usage of the removed APIs on Kubernetes 1.22 is found. More info: https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22
+//
 // Note that this validator allows to receive a List of optional values as key=values. Currently, only the
 // `k8s-version` key is allowed. If informed, it will perform the checks against this specific Kubernetes version where the
-// operator bundle is intend to be distribute for.
+// operator bundle is intend to be distribute for and will raise errors instead of warnings.
```

Comment:
I'm not sure what `intend to be distribute for` means. Can you elaborate further?

Reply:
I change the text. Please, let me know if it fits better.